fsspec/asyn.py

```diff
@@ -243,6 +243,9 @@ async def _pipe(self, path, value=None, **kwargs):
             *[self._pipe_file(k, v, **kwargs) for k, v in path.items()]
         )
 
+    async def _cat_file(self, path, start=None, end=None, **kwargs):
```

Comment:
@martindurant what's the purpose of `**kwargs` here? Should subclasses check that additional kwargs aren't passed?

Reply:
It's so that subclasses can pass any extra arguments (like headers) without changing the signature. If the subclass doesn't have any such thing it can do, then kwargs can be ignored.

Reply:
I guess its an indication to downstream that this is a reasonable place to add extra args, and it enables fsspec to add arguments (like start/end) without immediately braking other implementations for the common case that they are left with default values such as `None`.